# This is a combination of emg and imu data

Header header

uint32[8] emg

float64[3] angular_velocity

float64[3] linear_acceleration

float64[4] orientation # use quaternion coordinates
